.eslintrc

```diff
@@ -24,12 +24,12 @@
     "no-unused-vars": "warn"
   },
   "globals": {
-    "$": true,
-    "document": true,
-    "window": true,
-    "navigator": true,
-    "localStorage": true,
-    "jQuery": true,
+    "$": false,
```

Comment:
Curious as to why these are changed?

Reply:
If they're not required (browser-only) then you can remove them altogether.

Reply:
Because it's a node script, so you don't need any of the browsers feature, or jQuery

Reply:
Ah perfect, I'll remove them then!